chain/network/src/peer_manager/network_state/tier1.rs

```diff
@@ -66,13 +65,24 @@ impl super::NetworkState {
         futures_util::future::join_all(handles).await;
     }
 
+    /// Requests direct peers for accounts data full sync.
+    /// Should be called whenever the accounts_data.keys changes, and
+    /// periodically just in case.
+    pub fn tier1_request_full_sync(&self) {
+        self.tier2.broadcast_message(Arc::new(PeerMessage::SyncAccountsData(SyncAccountsData {
```

Comment:
do we have a metrics for this?

Reply:
We have metrics for messages sent,received and broadcasted by type.
I've added a dedicated metric for messages sent/received for each combination of values of `incremental` and `requesting_full_sync` fields.